packages/react-flicking/src/react-flicking/Flicking.tsx

```diff
@@ -138,6 +152,7 @@ class Flicking extends React.Component<Partial<FlickingProps & FlickingOptions>>
 
     sync(flicking, diffResult, this.reactPanels);
 
+    this._renderingState = RenderingState.UPDATED;
```

Comment:
If it is the same children even once, the renderingState is likely to be RENDERED forever.

Reply:
Good point.